### PR TITLE
move src/Billing to PSR12

### PR DESCRIPTION
**Pull request by stephenwaite**
another step in addressing #3392 

#### Short description of what this resolves:

cleans src/Billing

#### Changes proposed in this pull request:

fixes class names and method names/visibility etc to comply with PSR12

testing well

### CODE REVIEW

File: interface/billing/billing_process.php [hunk 1]
```diff
@@ -23,7 +23,8 @@
 
 use OpenEMR\Billing\BillingUtilities;
 use OpenEMR\Billing\Hcfa1500;
-use OpenEMR\Billing\X12_5010_837P;
+use OpenEMR\Billing\X125010837I;
+use OpenEMR\Billing\X125010837P;
```
Comment (bradymiller) [inline interface/billing/billing_process.php:27]: nice, they are starting to look like UUIDs (in the future, I predict all communication will be via UUIDs)
(bad joke)

Comment (stephenwaite) [reply interface/billing/billing_process.php:27]: :), they are something only a mother could love, but with another version in draft coming along, asc x12 7030 figured we could live with psr12 5010, somehow

Comment (stephenwaite) [reply interface/billing/billing_process.php:27]: should have mentioned that the `PSR4` move of Jerry's 837 is included

**Comment by bradymiller** [comment]
Code looks to me. If testing well, then bring it in :)

**Pull request merged**